home/cry.asm

```diff
@@ -0,0 +1,104 @@
+include "constants.asm"
+
+SECTION "Cry Home", ROM0 [$39b1]
+
+PlayStereoCry::
+	push af
+	ld a, $1
+	ld [wc1b9], a
+	pop af
+	jr asm_39c3
+
+PlayCry:: ; 00:39ba
+	push af
+	xor a
+	ld [wc1b9], a
+	ld [wc1ba], a
+	pop af
+asm_39c3: ; 00:39c3
```

Comment:
Is this actually used anywhere or is it just your disassembler trying to read opcodes where it shouldn't?

Reply:
Not sure what you're trying to ask here. These are most certainly the Pokémon cry-playing functions.

Unless you meant to place this comment elsewhere in the file?

Reply:
No, I was wondering if that label isn't useless.

Reply:
I direct you to compare [Crystal's functions.](/pret/pokecrystal/blob/master/home/cry.asm)